### PR TITLE
Solc5

### PR DESCRIPTION
**Pull request by nmclrn**
See commit messages.

Not published to npm yet. Should publish when merging.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,15 +14,16 @@
     "url": "git+https://github.com/TruSet/commit-reveal-voting.git"
   },
   "author": "Greg Taschuk <greg.taschuk@consensys.net> (https://github.com/gtaschuk) & Neil McLaren <neil.mclaren@consensys.net> (https://github.com/nmclrn)",
-  "license": "ALv2",
```
Comment (gtaschuk) [inline package.json]: please leave as is, this was jimjag's preferred license name

File: contracts/CommitRevealVoting.sol [hunk 2]
```diff
@@ -62,12 +62,12 @@ contract CommitRevealVoting {
     */
     function _commitVote(bytes32 _pollID, bytes32 _secretHash) internal
     {
-        require(commitPeriodActive(_pollID));
-        // prevent user from committing to zero node placeholder
-        require(_pollID != 0);
+        // prevent user from committing to a non-existent poll or one in the wrong state
+        require(_pollID != 0, "Not a valid poll");
```
Comment (gtaschuk) [inline contracts/CommitRevealVoting.sol]: splitting hairs but I would say that even more specifically this is Not a valid pollID

**Comment by gtaschuk** [comment]
tests are failing because the TestRBAC was removed

File: package.json (hunk 1 above)
Comment (nmclrn) [reply package.json]: @gtaschuk With the greatest respect to Jim, I think he got this one wrong. The string "AVLv2" is not on the list of licence names recognised by npm (see https://docs.npmjs.com/files/package.json#license and https://spdx.org/licenses/), which might be why https://www.npmjs.com/package/commit-reveal-voting still shows us using the MIT licence.

Out of interest, why did we use Apache for this repo but MIT for others? Should we change them all?

File: contracts/CommitRevealVoting.sol (hunk 2 above)
Comment (nmclrn) [reply contracts/CommitRevealVoting.sol]: Will change.

**Comment by nmclrn** [comment]
> tests are failing because the TestRBAC was removed

Weird that `truffle test` doesn't spot this failing migration. The RBAC is unused AFAICS so I'll just delete the migration.

**Comment by nmclrn** [comment]
Updates pushed, and published to npm as 1.1.0.

File: package.json (hunk 1 above)
Comment (nmclrn) [reply package.json]: > might be why https://www.npmjs.com/package/commit-reveal-voting still shows us using the MIT licence

It was. npm correctly shows Apache now that I published with the new string.

**Comment by nmclrn** [comment]
Pushed with RBAC added back in now. Not sure this matters to anyone but will re-publish as 1.1.1 for good measure.

File: package.json (hunk 1 above)
Comment (gtaschuk) [reply package.json]: We had opensourced bitmask-rbac before we spoke with Jim.

**Pull request merged**